### DISCUSSION TITLE
Gateway crash after a long uptime when accessing web interface

### DISCUSSION THREAD

**Discussion by Th0maz** [emsesp/EMS-ESP32#2034] [General]
I just want to share an experience I made a couple of days ago.
My S32 Gateway with release 3.6.5 has been running without errors for a long time.
I left it alone and it sent its values to an influxdb and responded to ping from my monitoring system.
When I then tried to access the web interface last week, the device immediately crashed and I had to powercycle it.

I have no further details or logs so far and did not investigate further. 
Maybe this makes sense to anybody? Or did anyone experience a similar case?

**Comment by proddy** [comment]
Thanks for reporting. There is an article on this at https://emsesp.org/Troubleshooting/#ems-esp-becomes-unresponsive

I don't think your gateway crashed. If the ESP32 mcu chip crashed it would have automatically rebooted without the need for a power-cycle. My guess it's WiFi related.
